packages/web/src/components/summary-table/SummaryTable.tsx

```diff
@@ -74,6 +99,30 @@ export const SummaryTable = ({
           </Text>
         </div>
       ) : null}
+    </>
+  )
+
+  const content = (
+    <div className={styles.container}>
+      <Text
```

Comment:
Text as div? mind explaining this for my learning purposes?

Reply:
Can't have divs in paragraphs, which is the default tag for `Text`

Reply:
Yes, what Marcus said. Also so that I can apply the row styles to it for flex/etc.